### PR TITLE
User

### PR DESCRIPTION
**Pull request by radhad-ux**
Structured User Class and implemented functionalities for User Controller/ Repo/Service classes

### CODE REVIEW

**Comment by sailathatammana** [comment]
Looks good

File: src/main/java/se/munchbox/user/UserService.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package se.munchbox.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service()
```
Comment (fnt90) [inline src/main/java/se/munchbox/user/UserService.java:8]: Should there be parenthesis here?

Comment (sailathatammana) [reply src/main/java/se/munchbox/user/UserService.java:8]: No, I belive.

Comment (radhad-ux) [reply src/main/java/se/munchbox/user/UserService.java:8]: Can you specify which line exactly? Is it line 8?

**Review by sailathatammana — APPROVED** [review]

File: src/main/java/se/munchbox/user/UserService.java (hunk 1 above)
Comment (fnt90) [reply src/main/java/se/munchbox/user/UserService.java:8]: Parentheses shouldn't be needed if it will be empty, as far as I know

Comment (radhad-ux) [reply src/main/java/se/munchbox/user/UserService.java:8]: Yes for Service annotation we dont need parantheses, but it doesnt give any error, anyway I will remove it

Comment (radhad-ux) [reply src/main/java/se/munchbox/user/UserService.java]: Deleted Parenthesis

**Pull request merged**